views/client.go

```diff
@@ -0,0 +1,90 @@
+package views
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/golang/glog"
+	livepeer "github.com/livepeer/go-api-client"
+	promClient "github.com/prometheus/client_golang/api"
+	prometheus "github.com/prometheus/client_golang/api/prometheus/v1"
+	"github.com/prometheus/common/model"
+)
+
+type TotalViews struct {
+	ID         string `json:"id"`
+	StartViews int64  `json:"startViews"`
```

Comment:
Does this need to be int64 or could it be uint32? Although at current present some videos on other platform have already overflowed uint32

Reply:
Prometheus returns a `float64` so I think it will be safer to just stay with the 64 bits as well. This is only on the API response, not being stored on any database anyway, so it should have any expensive consequences either. Would you have any other reason to prefer a 32 bit integer instead?